openssh/test/controls/service_running.rb

```diff
@@ -0,0 +1,14 @@
+control 'ensure sshd processes are all running and owned by correct owner' do
+
+  describe "openssh processes" do
```

Comment:
Is it possible to use the [`processes` resource](https://www.inspec.io/docs/reference/resources/processes/) here?

Reply:
Will investigate this.

Reply:
Updated test to use the 'processes' resource